airbyte-integrations/connectors/source-google-ads/setup.py

```diff
@@ -7,7 +7,7 @@
 
 # pin protobuf==3.20.0 as other versions may cause problems on different architectures
 # (see https://github.com/airbytehq/airbyte/issues/13580)
-MAIN_REQUIREMENTS = ["airbyte-cdk>=0.51.3", "google-ads==22.1.0", "protobuf", "pendulum"]
+MAIN_REQUIREMENTS = ["airbyte-cdk>=0.51.3", "google-ads==22.1.0", "protobuf", "pendulum>=3.0.0"]
```

Comment:
would not this cause any problem?

pendulum was pinned to `<3.0` in CDK
https://github.com/airbytehq/airbyte/pull/33572

Reply:
Pinned `<3.0.0` to align with the CDK